ci-operator/config/openshift/cluster-api-actuator-pkg/openshift-cluster-api-actuator-pkg-release-4.18__images.yaml

```diff
@@ -0,0 +1,28 @@
+base_images:
+  openshift_release_golang-1.21:
+    name: release
+    namespace: openshift
+    tag: golang-1.21
+build_root:
+  from_repository: true
+images:
+- dockerfile_path: Dockerfile.ci
+  inputs:
+    openshift_release_golang-1.21:
+      as:
+      - registry.ci.openshift.org/openshift/release:golang-1.21
+  to: cluster-api-actuator-pkg-test
```

Comment:
Should we use 1.22 https://github.com/openshift/cluster-api-actuator-pkg/blob/086923b9d3fd72a50a6c1587f8dc1189cae48df4/go.mod#L3 ?

Reply:
Updated, thank you.